apps/blog/src/api/query-helpers/post.ts

```
@@ -0,0 +1,37 @@
+      firstModDate: {
+        lte: new Date(), // I might finish the blog before the release date
+      },
+    } satisfies Prisma.PostWhereInput;
+  }
+
+  private static _includeWithMdx() {
+    return {
+      mdxContent: true,
+    } satisfies Prisma.PostInclude;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)